api/src/test/java/org/openmrs/validator/ConditionValidatorTest.java

```diff
@@ -14,11 +14,12 @@
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
-import org.openmrs.Cohort;
-import org.openmrs.Condition;
+import org.openmrs.*;
```

Comment:
Did you get a chance to look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports

Reply:
@dkayiwa I've just gone through it and making the necessary changes